docs/querying/basics.md

```diff
@@ -21,6 +21,37 @@ Other programs can fetch the result of a PromQL expression via the [HTTP API](ap
 This document is a Prometheus basic language reference. For learning, it may be easier to
 start with a couple of [examples](examples.md).
 
+## Samples
+
+The value of a sample at a given timestamp returned by PromQL may be a float or
+a [native histogram](https://prometheus.io/docs/specs/native_histograms).
+
+Native histograms can be optionally scraped by Prometheus or received by
+Prometheus over remote-write or OTLP.
```

Comment:
Floats can also be received over remote-write or OTLP.

The unique thing about NH is that you have to opt-in into scraping. Maybe we should not even mention that here (but make sure it is clearly called out in the scrape config). When querying, you don't care how a sample has entered the TSDB.